011_MultiplicationTable.js

```diff
@@ -0,0 +1,19 @@
+multiplicationtable = (n, index) => {
```

Comment:
It's not printed properly. Make sure you are doing everything properly. Don't just do it for the sake.